pandemania/src/App.jsx

```diff
@@ -33,6 +35,7 @@ function App() {
 							</Switch>
 						</div>
 					</Router>
+					<ToastContainer position="bottom-center" />
```

Comment:
Should it be lazy loaded?

Reply:
React.lazy doesn't support importing non-default components